tests/runfiles/linux.run

```diff
@@ -174,15 +174,14 @@ tests = ['zfs_send_001_pos', 'zfs_send_002_pos', 'zfs_send_003_pos',
 # DISABLED:
 # mountpoint_003_pos - needs investigation
 # ro_props_001_pos - https://github.com/zfsonlinux/zfs/issues/5201
-# readonly_001_pos - needs investigation
 # user_property_002_pos - needs investigation
 [tests/functional/cli_root/zfs_set]
 tests = ['cache_001_pos', 'cache_002_neg', 'canmount_001_pos',
     'canmount_002_pos', 'canmount_003_pos', 'canmount_004_pos',
     'checksum_001_pos', 'compression_001_pos', 'mountpoint_001_pos',
     'mountpoint_002_pos', 'reservation_001_neg',
     'share_mount_001_neg', 'snapdir_001_pos', 'onoffs_001_pos',
-    'user_property_001_pos', 'user_property_003_neg',
+    'user_property_001_pos', 'user_property_003_neg', 'readonly_001_pos'
```

Comment:
You're missing a comma after the test case and it's resulting in the [test being skipped](http://build.zfsonlinux.org/builders/Amazon%202015.09%20x86_64%20SIMD%20%28TEST%29/builds/2091/steps/shell_11/logs/stdio).